src/utils/clone.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Clone a given object, returning a disconnected copy.
+ *
+ * @note This should be replaced via StructuredClone in NodeJS 18.
+ * @note This only supports primative objects (array, object, string, etc.)
```

Comment:
typo: "primitive"

Reply:
Fixed in 37ddec60ec2777ac05267cf44bdbaeb2ef2e8c98